integration/test_collection_aggregate.py

```diff
@@ -17,55 +17,41 @@
 from weaviate.util import file_encoder_b64
 
 
-@pytest.fixture(scope="module")
-def client() -> Generator[weaviate.WeaviateClient, None, None]:
-    client = weaviate.connect_to_local()
-    client.collections.delete_all()
-    yield client
-    client.collections.delete_all()
-
-
 @pytest.mark.parametrize("how_many", [1, 10000, 20000, 20001, 100000])
-def test_collection_length(client: weaviate.WeaviateClient, how_many: int) -> None:
+def test_collection_length(
+    collection_factory: CollectionFactory, request: SubRequest, how_many: int
+) -> None:
```

Comment:
This pattern of using `request.node.name` as the collection name seems to be repeated everywhere. Can we move it into the fixture itself so that it only needs defining once _alla_ https://stackoverflow.com/a/58334351/14998213

Reply:
nice idea, done :)